### DISCUSSION TITLE
[DONE] Show machine in dashboard

### DISCUSSION THREAD

**Discussion by jone** [ParetoSecurity/pareto-mac#85] [Ideas]
It would be helpful to see some machine details in the team dashboard.
For instance the model (e.g. `MacBook Pro (13-inch, 2017, Two Thunderbolt 3 ports)`) and the serial number.
We have an inventory list with serial numbers and could then compare that and find/assign a device.

**Comment by zupo** [comment]
Sounds helpful, but let me think a bit about privacy implications.

**Reply by jone** [reply]
The target audience includes a lot of devs, at least at our company 😉 . I just thought about a feature (action in the menu for example) that just shows the current payload that would get sent over the wire.
This would build maximum transparency and at least allow users to inspect the data and speak up when they think that their privacy is in danger.

**Reply by zupo** [reply]
Ohhh that's a nice one!

**Reply by dz0ny** [reply]
Even better we can add a checkbox that will allow sending the "inventory" data. This way if the developer/employee is ok with sending that data it will be sent and shown otherwise not.

**Reply by dz0ny** [reply]
Deployed to Dash and App. Disabled by default.
![image](https://user-images.githubusercontent.com/239513/145287057-a8fa7c7f-b9ec-46b3-ae60-be4c54482aaf.png)

**Reply by zupo** [reply]
And this is how it looks in the Dashboard:

<img width="558" alt="Screenshot 2021-12-08 at 22 31 24" src="https://user-images.githubusercontent.com/311580/145301815-cb9956a2-ffd4-4636-8020-19f8ad9deda1.png">

**Reply by jone** [reply]
Very cool 🎉 thank you!
